docs(helm-values): add persistence.createStorageClass option (#1344)
longhorn/website: content/docs/1.12.0/references/helm-values.md

| persistence.backingImage.enable | bool | `false` | Setting that allows you to use a backing image in a Longhorn StorageClass. |
| persistence.backingImage.expectedChecksum | string | `nil` | Expected SHA-512 checksum of a backing image used in a Longhorn StorageClass. |
| persistence.backingImage.name | string | `nil` | Backing image to be used for creating and restoring volumes in a Longhorn StorageClass. When no backing images are available, specify the data source type and parameters that Longhorn can use to create a backing image. |

> | persistence.createStorageClass | bool | `true` | Setting that allows you to create the default Longhorn StorageClass ConfigMap. Set to false to skip StorageClass creation. |

Reviewer: ```suggestion
| persistence.createStorageClass | bool | `true` | Setting that enables creation of the default Longhorn StorageClass ConfigMap. Set to `false` to skip the creation. |
```

Author: Thanks for review. Agree
